libs/ui/src/lib/facet/list/list.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core'
+import { async, ComponentFixture, TestBed } from '@angular/core/testing'
+import { FormsModule } from '@angular/forms'
+
+import { ListComponent } from './list.component'
+
+describe('ListComponent', () => {
+  let component: ListComponent
+  let fixture: ComponentFixture<ListComponent>
+
+  const modelMock = {
```

Comment:
fixtures should be centralized, wait for my PR to be merged.

Reply:
I looked at your PR, but lib-ui depending on lib-search, only to be able to use fixtures in the unit tests, that feels strange to me.

Reply:
hmm yes ...
We could create a new folder `test` somewhere with all the fixtures...